R/sweep-clusters.R

```diff
@@ -1,16 +1,22 @@
 #' Calculate clusters across a set of parameters
 #'
-#' This function can be used to perform reproducible clustering while varying a set of parameters.
+#' @description This function can be used to perform reproducible clustering while varying a set of parameters.
 #' Multiple values can be provided for any of:
-#'  - The algorithm (`algorithm`)
-#'  - The weighting scheme (`weighting`)
-#'  - Number of nearest neighbors (`nn`)
-#'  - The resolution parameter (`resolution`)
-#'  - The objective function parameter (`objective_function`)
+#'
+#' - The algorithm (`algorithm`)
+#'
+#' - The weighting scheme (`weighting`)
+#'
+#' - Number of nearest neighbors (`nn`)
+#'
+#' - The resolution parameter (`resolution`)
+#'
+#' - The objective function parameter (`objective_function`).
```

Comment:
This formatting does not translate properly; we probably want to make all the docs in this package markdown format by adding `Roxygen: list(markdown = TRUE)` to the `DESCRIPTION` file